### ISSUE TITLE
Float error in QtScheduler in _start_timer

### ISSUE THREAD

**Issue by rvhu-cl** [agronholm/apscheduler#687]
### Things to check first

- [X] I have checked that my issue does not already have a solution in the [FAQ](https://apscheduler.readthedocs.io/en/master/faq.html)

- [X] I have searched the existing issues and didn't find my bug already reported there

- [X] I have checked that my bug is still present in the latest release

### Version

3.9.1.post1

### What happened?

get dump in _start_timer

self._timer = QTimer.singleShot(wait_time, self._process_jobs)
TypeError: arguments did not match any overloaded call:
  singleShot(int, PYQT_SLOT): argument 1 has unexpected type 'float'
  singleShot(int, Qt.TimerType, PYQT_SLOT): argument 1 has unexpected type 'float'

### How can we reproduce the bug?

program:

```python
from apscheduler.schedulers.qt import QtScheduler

def prompt(self):
    print("Ececuting Task....")

def add_scheduler(self):
    print("start_scheduler2")
    self.scheduler = QtScheduler()
    self.scheduler.add_job(self.prompt, 'interval', seconds=5)
    self.scheduler.print_jobs()
    self.scheduler.start()
```

when calling add_scheduler above float error dump.

**Comment by agronholm** [comment]
Could you include a fully runnable script that demonstrates the issue?

**Comment by rvhu-cl** [comment]
an other example is the default example.
"""
Demonstrates how to use the Qt compatible scheduler to schedule a job that executes on 3 second
intervals.
"""

from datetime import datetime
import signal
import sys

from apscheduler.schedulers.qt import QtScheduler

try:
    from PyQt5.QtWidgets import QApplication, QLabel
except ImportError:
    try:
        from PyQt4.QtGui import QApplication, QLabel
    except ImportError:
        try:
            from PySide6.QtWidgets import QApplication, QLabel
        except ImportError:
            try:
                from PySide2.QtWidgets import QApplication, QLabel
            except ImportError:
                from PySide.QtGui import QApplication, QLabel


def tick():
    label.setText('Tick! The time is: %s' % datetime.now())


if __name__ == '__main__':
    app = QApplication(sys.argv)

    # This enables processing of Ctrl+C keypresses
    signal.signal(signal.SIGINT, lambda *args: QApplication.quit())

    label = QLabel('The timer text will appear here in a moment!')
    label.setWindowTitle('QtScheduler example')
    label.setFixedSize(280, 50)
    label.show()

    scheduler = QtScheduler()
    scheduler.add_job(tick, 'interval', seconds=3)
    scheduler.start()

    # Execution will block here until the user closes the windows or Ctrl+C is pressed.
    app.exec_()

after running this I get the error.
python3 scheduler.py
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Traceback (most recent call last):
  File "/home/therol6/.local/lib/python3.10/site-packages/apscheduler/schedulers/qt.py", line 50, in _process_jobs
    self._start_timer(wait_seconds)
  File "/home/therol6/.local/lib/python3.10/site-packages/apscheduler/schedulers/qt.py", line 37, in _start_timer
    self._timer = QTimer.singleShot(wait_time, self._process_jobs)
TypeError: arguments did not match any overloaded call:
  singleShot(int, PYQT_SLOT): argument 1 has unexpected type 'float'
  singleShot(int, Qt.TimerType, PYQT_SLOT): argument 1 has unexpected type 'float'

**Comment by agronholm** [comment]
Which QT Python bindings library were you using (and what version)?

**Comment by agronholm** [comment]
The example runs fine for me on PySide6.

**Comment by rvhu-cl** [comment]
I'm using PYQT5 because PySide6 is not running correct on raspberry pi

**Comment by enrico5** [comment]
`util.timedelta_seconds` returns a float.

This value is returned by `BaseScheduler._process_jobs` and stored into wait_seconds, still as a float.

`wait_time` is still a float and passed as the first argument of `QTimer.singleShot`, which expects an int (also by PySide6).

There should be a conversion to int somewhere. It's lucky that it works in some cases.

**Comment by agronholm** [comment]
Closing as this was fixed in v3.10.1.
